README.md

```diff
@@ -23,12 +26,19 @@ This project is configured using environment variables.
 | Variable             | Description                                                                        | Default                      |
 | -------------------- | ---------------------------------------------------------------------------------- | ---------------------------- |
 | `SECRET_KEY`         | Django Secret Key                                                                  | `### CHANGE ME ###`          |
+| `DEBUG`              | Django Debug mode                                                                  | False                        |
 | `DB_ENGINE`          | Database Engine, set to `django.db.backends.postgresql_psycopg2` to use PostgreSQL | `django.db.backends.sqlite3` |
 | `DB_NAME`            | Database Name                                                                      | `db.sqlite3`                 |
 | `DB_HOST`            | Database Hostname, set this when using PostgreSQL                                  | _empty string_               |
 | `DB_USER`            | Database Username, set this when using PostgreSQL                                  | _empty string_               |
 | `DB_PASS`            | Database Password, set this when using PostgreSQL                                  | _empty string_               |
 | `DB_PORT`            | Database Port, set this when using PostgreSQL                                      | `5432`                       |
+| `OAUTH_CLIENT_ID`    | OAuth authentication client id (found in observation portal admin for the app)     | ``                           |
```

Comment:
Not sure who the target audience is for the readme, but it's not clear to me where I would find this value. Is it defined when creating the oauth server somewhere else? If it's something a new user trying to set things up (me) should be able to figure out, then additional details might be helpful.

Reply:
The oauth client id and secret and autogenerated for you when you create an oauth "application" on the server. We should probably describe this better in overall project docs and reference from here...

Reply:
Agreed, when we work out how we want to do oauth across the projects, I think that would be a good time to create some documentation on it.